lib/web/scripts/node-join/install.sh

```diff
@@ -449,6 +450,16 @@ proxy_service:
   enabled: no
 EOF
 }
+# get the auth section of a node config
+get_node_auth_config() { 
+    if [ ${JOIN_METHOD} == "iam" ]; then 
```

Comment:
I can never remember where single brackets are okay. IIRC single brackets need quoted vars. Maybe just use double brackets to be sure?

```suggestion
    if [[ ${JOIN_METHOD} == "iam" ]]; then 
```